src/components/Todo/Todo.scss

```
@@ -0,0 +1,18 @@
+.todo {
+  min-width: 360px;
+  max-width: 500px;
+  margin: auto;
+  padding: 25px;
+
+  &__title {
+    font-weight: bold;
+
+    &_completed::before {
+      content: "DONE";
```

```suggestion
      content: "done";
```
use text-transform property to make text uppercased